index.html

```diff
@@ -287,6 +290,12 @@ <h2 id='stipends'>Stipends</h2>
                 with a short letter of support.
                 </p>
 
+                <h2>Submissions</h2>
+                We are soliciting ideas for presentations of roughly 10-20 minutes each.  
+                To submit an idea for a talk, please send by <strong>April 15</strong> a short abstract to 
```

Comment:
better to link the words "send a short abstract" instead of spelling out the email?